test/system/spam2_test.rb

```diff
@@ -33,6 +33,13 @@ def setup
     page.assert_selector("div.alert", text: "Content spammed")
   end
 
+  test "unflag post in spam2" do
```

Comment:
This is great too! Do you think it'd be possible to add a couple for regular users flagging posts, and for some other specific review action by moderators?

Reply:
@jywarren I have added a system test where user can mark node as a flag. Please review Thanks!!